datacapturing/src/main/java/de/cyface/datacapturing/ui/CapturingNotification.java

```diff
@@ -18,11 +18,17 @@
  *
  * @author Klemens Muthmann
  * @author Armin Schnabel
- * @version 2.1.0
+ * @version 2.1.1
  * @since 1.0.0
  */
 public class CapturingNotification {
 
+    /**
+     * Identifies the capturing ongoing {@link android.app.Notification} which can be implemented by sdk using apps.
+     * These ids should be placed centrally to avoid duplicate entries.
+     */
+    public final static int CAPTURING_NOTIFICATION_ID = 1;
```

Comment:
Wollten wir es nicht dem User überlassen, das hier zu übergeben?

Reply:
Never mind. This is the legacy way for the old notification id, I see! Maybe add a todo, so I do not get confused.